README.md

```diff
@@ -3,9 +3,9 @@
 ## On Windows
 
 ```bash
-docker build -t ros_base_station:latest .
-export DISPLAY=128.180.246.10:0.0 # replace with your IP address
-docker run -it -e DISPLAY=$DISPLAY --net=host ros_base_station:latest
+docker build -t urc_software .
+export DISPLAY=192.168.4.49:0.0 # replace with your IP address up to the colon

```

Comment:
Didn't we say we can configure it so we can pass the ip in the docker run cmd and don't need to put this in the build file?

Reply:
Are you talking about when we run docker like this for example? docker run -it -e DISPLAY=128.180.246.10:0.0 --net=host urc_software

This doesn't work for me. I have to export it in my git bash then run docker run

Reply:
Well I think there's a way it should work. Also I'm pretty sure --host network doesn't work on windows. Anyway we need to find a solution where this is not in the build file so git is not tracking these temporary IPs.